src/server/editorServices.ts

```diff
@@ -411,6 +411,7 @@ namespace ts.server {
         /** @deprecated use serverMode instead */
         syntaxOnly?: boolean;
         serverMode?: LanguageServiceMode;
+        session: Session<unknown> | undefined;

```

Comment:
Please add type parameters to the options as well as Project service please with same default as session <TMessage = string>`